### ISSUE TITLE
Images with dask-gateway 0.9

### ISSUE THREAD

**Issue by scottyhq** [pangeo-data/pangeo-cloud-federation#859]
@TomAugspurger -just noting that docker images containing dask-gateway=>0.9 will currently fail to spin up clusters (Pangeo docker images > 2020.11.06)
https://github.com/pangeo-data/pangeo-docker-images/blob/2020.11.06/pangeo-notebook/packages.txt#L77

pod logs show:
```
 kubectl logs -n  dask-worker-e5b20696082141d4840cf40377a5328f-4g27g  
/srv/start: line 12: exec: dask-gateway-worker: not found
```

If I understand correctly we need to bump the [daskhub chart](https://github.com/dask/helm-chart) version to 4.5.4?
https://github.com/pangeo-data/pangeo-cloud-federation/blob/b512e11dfb185a3fe65af7ed208607431d93846c/pangeo-deploy/Chart.yaml#L8

**Comment by scottyhq** [comment]
We can use this issue to track the version bump in #875 

@tjcrone and @TomAugspurger - looks like things did not deploy smoothly on either GCP or OOI:
https://app.circleci.com/pipelines/github/pangeo-data/pangeo-cloud-federation/1622/workflows/effd2152-bae4-4722-8ffb-7cedfada6bed/jobs/1789 

AWS Deployed via github actions and pods are running, but I'm unable to access staging:

<img width="838" alt="Screen Shot 2020-11-24 at 3 39 23 PM" src="https://user-images.githubusercontent.com/3924836/100164039-e303fd80-2e6b-11eb-8c22-72ffd83c8a8a.png">

I see "Not Secure" on the landing page, so I think something happened with HTTPS

**Comment by scottyhq** [comment]
closed by #895
